Sources/SwifterSwift/SwiftStdlib/MutableCollectionExtensions.swift

```diff
@@ -22,3 +22,19 @@ public extension MutableCollection where Self: RandomAccessCollection {
         sort { $0[keyPath: keyPath] < $1[keyPath: keyPath] }
     }
 }
+
+public extension MutableCollection {
+    /// SwifterSwift: Assign to an given value to a field of all elements of the collection
+    ///
+    /// - Parameter value: The new value of the field
+    /// - Parameter keyPath: The actual field of the element
+    mutating func assignToAll<Value>(value: Value, by keyPath: WritableKeyPath<Element, Value>) {
+        guard !isEmpty else { return }
+
+        var it = startIndex
```

Comment:
renaming `it` to `index` should fix the swiftlint warning

Reply:
okay, I renamed it to `ind` though, as it was having a name collision

Reply:
Maybe `idx`?